### PR TITLE
[GREENHOST-198] fix broken markup and progress bar

### PR DESCRIPTION
**Pull request by ihor-romaniuk**
**Description:**  fix broken markup and progress bar
**Youtrack:** https://youtrack.raccoongang.com/issue/GREENHOST-198

### CODE REVIEW

File: video_xblock/static/vendor/js/Vimeo.js [hunk 1]
```diff
@@ -390,8 +397,8 @@ THE SOFTWARE. */
               '.vjs-vimeo .vjs-iframe-blocker { display: none; }' +
               '.vjs-vimeo.vjs-user-inactive .vjs-iframe-blocker { display: block; }' +
               '.vjs-vimeo .vjs-poster { background-size: cover; }' +
-              // '.vjs-vimeo { height:100%; }' +
-              '.vimeoplayer { width:100%; height:180%; position:absolute; left:0; top:-40%; }';
+              '.vimeoplayer { width:100%; height:100%; position:absolute; left:0; top:0; pointer-events: none;}' +
+              '.video-js .vjs-control-bar { z-index: 3 }';
```
Comment (NikolayBorovenskiy) [inline video_xblock/static/vendor/js/Vimeo.js]: z-index just a three. Do you think this is a robust way? Does it make sense to increase the value? As for me I usually set custom z-index pretty big to be sure that my custom always be on a top 🙂

File: video_xblock/static/vendor/js/Vimeo.js [hunk 2]
```diff
@@ -156,28 +156,36 @@ THE SOFTWARE. */
...
     onFinish: function() {
       this.vimeoInfo.state = VimeoState.ENDED;
       this.trigger('ended');
     },
+
     onSeek: function(data) {
       this.trigger('seeking');
       this.vimeoInfo.time = data.seconds;
+      this.vimeoInfo.duration = data.duration;
+      this.trigger("durationchange");
```
Comment (NikolayBorovenskiy) [inline video_xblock/static/vendor/js/Vimeo.js]: I see you find a workaround for the headache. Nice if it works well. But it looks like as a workaround🙂

File: video_xblock/static/vendor/js/Vimeo.js (hunk 1 above)
Comment (ihor-romaniuk) [reply video_xblock/static/vendor/js/Vimeo.js]: I decided that in this case, it's sufficient value.

File: video_xblock/static/vendor/js/Vimeo.js (hunk 2 above)
Comment (ihor-romaniuk) [reply video_xblock/static/vendor/js/Vimeo.js]: In this case, I use callback function from videoJS library which we use for render video player

**Pull request merged**